modules/extensions-support/src/main/java/org/mule/runtime/module/extension/internal/runtime/operation/OperationMessageProcessor.java

```diff
@@ -89,28 +107,54 @@
 
   private ExecutionMediator executionMediator;
   private OperationExecutor operationExecutor;
+  private PolicyManager policyManager;
   protected ReturnDelegate returnDelegate;
 
   public OperationMessageProcessor(ExtensionModel extensionModel,
                                    OperationModel operationModel,
                                    ConfigurationProvider configurationProvider,
                                    String target,
                                    ResolverSet resolverSet,
-                                   ExtensionManagerAdapter extensionManager) {
+                                   ExtensionManagerAdapter extensionManager,
+                                   PolicyManager policyManager) {
     super(extensionModel, operationModel, configurationProvider, extensionManager);
     this.extensionModel = extensionModel;
     this.operationModel = operationModel;
     this.resolverSet = resolverSet;
     this.target = target;
     this.entityMetadataMediator = new EntityMetadataMediator(operationModel);
+    this.policyManager = policyManager;
   }
 
   @Override
   public Event process(Event event) throws MuleException {
-    return (Event) withContextClassLoader(getExtensionClassLoader(), () -> {
+    return withContextClassLoader(getExtensionClassLoader(), () -> {
```

Comment:
This method too, although not as bad as the other one could do with refactoring/shortening. By pulling about the policy lookup/application into other methods maybe?